### PR TITLE
Update 1.10.6

### PR DESCRIPTION
**Pull request by ocefpaf**
Required in https://github.com/conda-forge/descarteslabs-feedstock/pull/55

### CODE REVIEW

**Comment by ocefpaf** [comment]
@hmaarrfk I'm getting a segfault here. Do you think your changes in #40 could help? I can port them over.

**Comment by hmaarrfk** [comment]
Maybe. I feel like the blocs feedstock is tested

**Comment by hmaarrfk** [comment]
it might also be due to emulation. 

it happens with alot of tuned algorithms

**Comment by ocefpaf** [comment]
> it might also be due to emulation.
> 
> it happens with alot of tuned algorithms

Looks like that is it. Two patches are already in this release and I added the only one that is not. @hmaarrfk, do you mind taking a quick look to ensure I did not make any mistakes porting your changes from #40?

**Comment by ocefpaf** [comment]
@hmaarrfk I'm not sure what else I can do to fix aarch64. Any advice? Would it be OK to merge without it and opening an issue? I want to unlock some downstream package that requires this version.

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

File: conda-forge.yml [hunk 1]
```diff
@@ -1,11 +1,10 @@
 bot:
   automerge: true
 build_platform:
-  linux_aarch64: linux_64
   osx_arm64: osx_64
 conda_forge_output_validation: true
 provider:
-  linux_aarch64: default
+  linux_aarch64: azure
```
Comment (ocefpaf) [inline conda-forge.yml:7]: I misinterpreted your comment and thought the default, no emulation, was preferred to fix this. Thanks!

**Comment by hmaarrfk** [comment]
Cross compiling is tricky

**Pull request merged**